flake.nix

```
@@ -374,15 +388,42 @@
           '';
         };
 
-        git-hooks-nix = {
-          baseUrl = "https://github.com/cachix/git-hooks.nix/blob/master";
+        pkgs-by-name-for-flake-parts = {
+          title = "pkgs-by-name-for-flake-parts";
+          baseUrl = "https://github.com/drupol/pkgs-by-name-for-flake-parts";
```

```suggestion
          baseUrl = "https://github.com/drupol/pkgs-by-name-for-flake-parts/blob/master";
```

This is weird. I should refactor this.

---

Do you need me to fixup those commits?